app/build.gradle.kts

```diff
@@ -85,12 +99,15 @@ android {
     }
 
     kotlinOptions {
-        jvmTarget = "1.8"
+        jvmTarget = JavaVersion.VERSION_11.toString()
```

Comment:
I like the following syntax better:
jvmTarget = "${JavaVersion.VERSION_11}"
What do you think?

Reply:
From what I've researched there is no notable difference from jvmTarget 1.8 to higher. But I think it's a good idea to use the latest one.